src/bp/ui-studio/src/web/components/Confusion/index.jsx

```diff
@@ -0,0 +1,114 @@
+import { FaThLarge } from 'react-icons/fa'
+import React from 'react'
+import classNames from 'classnames'
+import axios from 'axios'
+import style from './style.scss'
+
+export default class NluPerformanceStatus extends React.Component {
+  state = {
+    health: 'gray',
+    botId: undefined
+  }
+  status = {
+    f1score: null,
+    unsynced: false,
+    computing: false
+  }
```

Comment:
Shouldn't this be in the state ?

Reply:
It doesn't need to be in the state, because it has nothing to do with the view.